.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# To use these pre-commit hooks you should first install https://pre-commit.com
+# and then run "pre-commit install" in the root folder of your local clone
+# so the hooks are deployed to .git/hooks
+#
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+#
+# the pep8 custom config (e.g. custom line lenght) is defined in file .pep8
+# https://pep8.readthedocs.io/en/release-1.7.x/intro.html#configuration
+
+repos:
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    sha: v1.2.0
+    hooks:
+      - id: autopep8-wrapper
+        name: 'autopep8-wrapper (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: check-ast
+        name: 'check-ast (.eb)'
+        types: [file]
+        files: \.eb$
+      - id: autopep8-wrapper
+      - id: check-ast
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-merge-conflict
+
+  - repo: https://github.com/pescobar/pre-commit-hooks-easybuild
+    sha: v0.1
```

Comment:
v0.1 is not a SHA?

Reply:
It's an unfortunate mistake, this is correct though (and tags are preferred here).

When I initially made the framework it only worked with sha1 references. It's confusing, but difficult to change (the oldest open issue in our tracker is to migrate this)

Reply:
An update on this btw, the latest version of pre-commit supports `rev` as well as `sha` (originally part of [v1.7.0](https://github.com/pre-commit/pre-commit/releases/tag/v1.7.0))